topdown/cache/cache_test.go

```diff
@@ -35,6 +41,10 @@ func TestParseCachingConfig(t *testing.T) {
 			input:   []byte(`{"inter_query_builtin_cache": {},}`),
 			wantErr: false,
 		},
+		"default_num_entries": {
+			input:   []byte(`{"regex_builtin_cache": {},}`),
+			wantErr: false,
+		},
```

Comment:
Should the glob cache also get asserted?

Reply:
Yes, more tests are coming 😄